collect_app/src/main/java/org/odk/collect/android/activities/CaptureSelfieActivity.java

```diff
@@ -90,6 +105,11 @@ private void savePhoto(byte[] data) {
         }
     }
 
+    /**
+     * Get an available front {@link Camera} instance, and do some initialization for it.
```

Comment:
Doc comments should use the indicative verb mood. In this case get -> gets, do -> does.